spynnaker/pyNN/models/spike_source/spike_source_poisson_vertex.py

```diff
@@ -191,7 +194,7 @@ def get_resources_used_by_atoms(
             [self._spike_recorder.get_sdram_usage_in_bytes(
                 vertex_slice.n_atoms, self._max_spikes_per_ts(
                     vertex_slice, n_machine_time_steps, machine_time_step),
-                self._N_POPULATION_RECORDING_REGIONS) * n_machine_time_steps],
+                n_machine_time_steps)],
```

Comment:
If _N_POPULATION_RECORDING_REGIONS is no longer being used then it should be deleted at the top of the class...

Reply:
Indeed it should - fixed!